features/other_a/languages.feature

```diff
@@ -1,5 +1,32 @@
 @works
 Feature: Languages
+
+  Scenario: Add a language and check abuse and support available
```

Comment:
Oh! It looks like we have a test for adding abuse support for a language: https://github.com/otwcode/otwarchive/blob/d1858d91cfacd9b6c23d3079538bd61db2859e44/features/admins/admin_languages.feature 

Maybe just add a test for support available to that file?